### PR TITLE
adding-publicity header

### PR DESCRIPTION
**Pull request by JaqKent**

### CODE REVIEW

File: src/components/PublicityHeader/constants.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import payment from 'assets/payment-method.svg';
+import articulo1 from 'assets/articulo1.jpg';
+import articulo2 from 'assets/articulo2.jpg';
+
+export const SERVICES = [
+  {
+    id: 1,
+    image: store,
+    text1: 'PUNTO DE RETIRO',
+    text2: 'Mas información',
```
Comment (nathan-hyan) [inline src/components/PublicityHeader/constants.ts:12]: Text can be moved to a constant and reused in all places

File: src/components/PublicityHeader/index.tsx [hunk 2]
```diff
@@ -1,16 +1,36 @@
...
   return (
-    <div>
-      <Slider hide auto={auto} setAuto={setAuto} slides={ITEMS} className={styles.image} />
+    <>
+      <div>
+        <Slider hide auto={auto} setAuto={setAuto} slides={ITEMS} className={styles.image} />
+      </div>
+      <div className={styles.services}>
+        {SERVICES.map((service) => (
+          <div className={styles.iconsSection}>
```
Comment (nathan-hyan) [inline src/components/PublicityHeader/index.tsx:17]: Move this block of code to a new component

**Review by nathan-hyan — COMMENTED** [review]
Great work

File: src/components/PublicityHeader/constants.ts (hunk 1 above)
Comment (JaqKent) [reply src/components/PublicityHeader/constants.ts:12]: no need, its never gonna be used again, never

File: src/components/PublicityHeader/index.tsx (hunk 2 above)
Comment (JaqKent) [reply src/components/PublicityHeader/index.tsx:17]: done

**Review by nathan-hyan — APPROVED** [review]

**Pull request merged**